server/text_generation_server/utils/logits_process.py

```diff
@@ -30,6 +30,7 @@ def __init__(
         top_k=None,
         top_p=None,
         typical_p=None,
+        no_repeat_ngram_size=None,
```

Comment:
`no_repeat_ngram_size` is now passed to `StaticWarper` but it doesn't seem like any new warpers that make use of the new argument. Do we need  `NoRepeatNGramLogitsProcessor` appended to `self.warpers`?

Reply:
ah good catch!
Actually there's one commit missing here from the original PR, the `NoRepeatNGramLogitsProcessor` shouldn't be in the warpers in the first place.

Reply:
now there shouldn't be things that changed the `StaticWarper` 👍  only stylistic changes